packages/yourloops/test/integration/assert/trends.ts

```diff
@@ -48,14 +50,14 @@ export const checkTrendsTidelineContainerTooltips = async () => {
   userEvent.unhover(cbgSlice)
 }
 
-export const checkTrendsStatsWidgetsTooltips = () => {
+export const checkTrendsStatsWidgetsTooltips = async () => {
   const statsWidgets = within(screen.getByTestId('stats-widgets'))
-  checkStatTooltip(statsWidgets, 'Avg. Daily Time In Range', TIME_IN_RANGE_TOOLTIP)
-  checkStatTooltip(statsWidgets, 'Avg. Glucose (CGM)', AVG_GLUCOSE_TOOLTIP)
-  checkStatTooltip(statsWidgets, 'Sensor Usage', SENSOR_USAGE_TOOLTIP)
-  checkStatTooltip(statsWidgets, 'GMI (estimated HbA1c)', GMI_TOOLTIP)
-  checkStatTooltip(statsWidgets, 'Standard Deviation', STANDARD_DEVIATION_TOOLTIP)
-  checkStatTooltip(statsWidgets, 'CV (CGM)', CV_TOOLTIP)
+  await checkStatTooltip(statsWidgets, 'Avg. Daily Time In Range', TIME_IN_RANGE_TOOLTIP)
```

Comment:
await is due to MUI tooltip right ? waiting for it to disapear ?

Reply:
yes